div-icon.js

```diff
@@ -46,7 +46,12 @@ export default class Divicon extends MapLayer {
   }
 
   updateLeafletElement(fromProps, toProps) {
-    if (toProps.position !== fromProps.position) {
+    // Even if we don't get a legnth of 2, this will work as the array will return undefined
+    const fromPosition = fromProps.position || []
```

Comment:
The `fromPosition` and `toPosition` variables in the updateLeafletElement method are declared but never used.

Reply:
Sorry for the late reply, I don't know if it is still relevant but fixed it. Was suppose to use them for the comparison.